DMTR-201.tex

```diff
@@ -0,0 +1,2058 @@
+\documentclass[DM,lsstdraft,STR,toc]{lsstdoc}
+\usepackage{geometry}
+\usepackage{longtable,booktabs}
+\usepackage{enumitem}
+\usepackage{arydshln}
+
+\input meta.tex
+
+\providecommand{\tightlist}{
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
+\begin{document}
+
+\def\milestoneName{Fall 2019 Pipelines Release Acceptance Test Campaign}
+\def\milestoneId{LVV-P65}
+\def\product{Acceptance}
+
+\setDocCompact{true}
+
+\title{ LVV-P65 Fall 2019 Pipelines Release Acceptance Test Campaign Test Plan and Report}
+\setDocRef{\lsstDocType-\lsstDocNum}
+\date{\vcsdate}
+\setDocUpstreamLocation{\url{https://github.com/lsst/lsst-texmf/examples}}
+\author{ Jeffrey Carlin }
+
+\input history_and_info.tex
+
+
+\setDocAbstract{
+This is the test plan and report for LVV-P65 (Fall 2019 Pipelines Release Acceptance Test Campaign),
+an LSST level 2 milestone pertaining to the Data Management Subsystem.
+}
+
+
+\maketitle
+
+\section{Introduction}
+\label{sect:intro}
+
+
+\subsection{Objectives}
+\label{sect:objectives}
+
+This Acceptance Test campaign aims to verify a small number of
+\href{https://lse-61.lsst.io/}{DMSR} (\citeds{LSE-61}) requirements related to
+the LSST Science Pipelines. It will be executed in conjunction with the
+release of Science Pipelines Version 19.0.0, but the pipeline release is
+not contingent upon this test campaign. This Test Plan aims to
+demonstrate that the included requirements have been met by Version
+19.0.0 of the Pipelines, and to thus fully verify their completion and
+readiness for LSST Operations.
+
+
+
+\subsection{System Overview}
+\label{sect:systemoverview}
+
+The tests to be executed are intended to verify that the DM system
+satisfies a subset of the requirements outlined in the Data Management
+System Requirements (DMSR; \href{https://lse-61.lsst.io/}{\citeds{LSE-61}} ).
+This subset of requirements is related to pipeline algorithms, and was
+selected for this campaign to coincide with the release of a new version
+of the LSST Science Pipelines. Additional DMSR requirements will be
+verified in later Acceptance Test
+Campaigns.\\[2\baselineskip]\textbf{Applicable
+Documents:}\\[2\baselineskip]\citeds{LSE-61} Data Management System
+Requirements\\
+\citeds{LDM-503} Data Management Test Plan\\[2\baselineskip]The tests will be
+performed using the HSC-RC2 dataset (as defined in
+\href{https://jira.lsstcorp.org/browse/DM-11345}{DM-11345} ). When
+possible, we will start our tests with the data products resulting from
+processing HSC-RC2 with the w\_2019\_46 weekly pipelines release (
```

Comment:
The HSC-RC2 processing that is on lsst-dev was done in repos tagged with weeklies. So I meant that choosing the w_2019_46 RC2 processing is the same as saying "v19" (since that's the weekly that became v19). I can clarify this in the document.